### ISSUE TITLE
Specifying incorrect dependency doesn't issue a warning

### ISSUE THREAD

**Issue by chriseidhof** [sbt/sbt#73]
When you specifiy a non-existing dependency (e.g. by making a typo in the jar name), and run sbt update, you don't get any feedback on that. Especially for newbies this is hard, because you have no clue what the problem is.

**Comment by harrah** [comment]
Please specify enough information to reproduce.  What kind of dependency?  A non-existing managed dependency like `"a" % "a" % "1.0"` gives an error message like `a#a;1.0: not found`.

**Comment by ijuma** [comment]
I helped Chris on this in #scala and the situation is not as simple. If the dependency doesn't exist, then an error _is_ issued. In this case, the dependency exists, but is of type pom:

http://repo1.maven.org/maven2/org/twitter4j/twitter4j/2.2.3/twitter4j-2.2.3.pom

I can't think of a situation where specifying this kind of dependency makes sense (although I could be missing something) and hence why I suggested a warning.
